integration-tests/NAME.IntegrationTests/Constants.cs

```diff
@@ -31,5 +36,9 @@ public static class Constants {
         SpecificMongoVersion = Environment.GetEnvironmentVariable("SPECIFIC_MONGO_VERSION") ?? "3.0.0";
         SpecificRabbitVersion = Environment.GetEnvironmentVariable("SPECIFIC_RABBIT_VERSION") ?? "3.6.5";
         SpecificServiceVersion = Environment.GetEnvironmentVariable("SPECIFIC_SERVICE_VERSION") ?? "1.0.0";
+
+        LatestElasticHostname = Environment.GetEnvironmentVariable("LATEST_ELASTIC_HOSTNAME") ?? "localhost";
```

Comment:
The Environment Variables should also contain the full name of Elasticsearch, should be `LATEST_ELASTICSEARCH_HOSTNAME`